custom-vu/src/main/kotlin/jces1209/vu/JiraCloudScenario.kt

```diff
@@ -63,17 +61,6 @@ class JiraCloudScenario : Scenario {
                 jira = jira,
                 meter = meter,
                 projectMemory = similarities.projectMemory
-            ),
-            browseBoards = BrowseCloudBoards(
-                jira = jira,
-                meter = meter,
-                boardsMemory = boardPages
-            ),
-            viewBoard = ViewCloudBoard(
-                jira = jira,
-                meter = meter,
-                boardMemory = boardPages,
-                issueKeyMemory = similarities.issueKeyMemory
```

Comment:
This is amazing. I didn't expect we'd be able to reduce both actions. We can not see that there isn't that much difference in how boards are tested. Only the pages are different, but the flow is the same. 🚀